API.Microsoft.Extensions.DependencyInjection.AutoActivation/WithDocs/Microsoft.Extensions.DependencyInjection/AutoActivationExtensions.cs

```diff
@@ -0,0 +1,293 @@
+// Assembly 'Microsoft.Extensions.DependencyInjection.AutoActivation'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+/// <summary>
+/// Extension methods for automatically activating singletons after application starts.
+/// </summary>
+public static class AutoActivationExtensions
+{
+    /// <summary>
+    /// Enforces singleton activation at startup time rather then at runtime.
```

Comment:
Can we add a remark that this method and other overloads have a prerequisite that a singleton is already registered in DI?